src/firmware/main/primitives/jcatch.c

```diff
@@ -43,24 +43,23 @@ static primitive_params_t catch_param;
  */
 static void catch_init(void) {}
 
+
+float catchvelocity;  // 0.4
```

Comment:
What are the commented decimals for?

Reply:
No idea, leaving as out-of-scope for now though, will be focusing on the primitives in a PR in the very near future.

Reply:
No idea, leaving as out-of-scope for now though, will be focusing on the primitives in a PR in the very near future.